avro_schema/frontend.lua

```diff
@@ -135,6 +135,24 @@ local copy_schema
 local copy_schema_error
 local copy_schema_location_info
 
+-- add new type definition to scope
```

Comment:
Side note:

What with recursive definitions? We cannot use it with flatten / unflatten AFAIR, but with validate we cannot too?